nexus-tweaks.user.js

```diff
@@ -1144,6 +1144,43 @@ promiseList.push((async () => {
 })());
 
 
+//##############################################################################
+promiseList.push((async () => {
+  const mod = await nexusTweaks.registerModule(
+    'lichDelight',
+    'Lich Pet Summon fix',
+    'local',
+    'Fixes the UI for summoning lich pets. Should make the pet list more eye-friendly.',
+  );
+
+  const lichDelight = () => {
+    'use strict';
+
+    const petTableTable = document.querySelector('.petTable');
+    if (!petTableTable) return;
+
+    // Pets summon buttons
+    const petSummons = petTableTable.querySelector('tbody').querySelector('table.summonsButtons>tbody');
+    const FM = petSummons.querySelector('input[value="Fossil Monstrosity"]');
+    const necro = petSummons.querySelector('input[value="Necrophage"]');
+
+    if (FM) {
+      FM.nextSibling.value = 'Fossil Monstrosity (3 Skeletons) '
+      if (petSummons.children.length < 3) petSummons.appendChild(document.createElement('tr'));
+      petSummons.children[2].insertBefore(FM.parentNode.parentNode, petSummons.children[2].firstChild);
```

Comment:
When you're manipulating DOM elements with relative positions like this, it helps a lot to document what you expect those elements to be.